src/components/index.ts

```diff
@@ -1 +1,4 @@
 export { default as Router } from './router/router'
+export { default as Button } from './button/button'
+export { default as Input } from './input/input'
+export { default as Select } from './select/select'
```

Comment:
Quando você criar o index.ts pra cada pasta, troca esses importa para 
```suggestion
export { default as Button } from './button'
export { default as Input } from './input'
export { default as Select } from './select'
```

Reply:
commit: a694929fa602d9e2540b7bd3ca5bcbb3be2f9890